src/test/java/de/uhd/ifi/se/decision/management/jira/consistency/TestDuplicateTextDetector.java

```diff
@@ -0,0 +1,70 @@
+package de.uhd.ifi.se.decision.management.jira.consistency;
+
+import com.atlassian.jira.issue.MutableIssue;
+import com.atlassian.jira.project.Project;
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.BasicDuplicateTextDetector;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.DuplicateFragment;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraIssues;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraProjects;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class TestDuplicateTextDetector extends TestSetUp {
+	private static MutableIssue baseIssue, otherIssue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)